site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -256,6 +257,65 @@ image::installation-guides/cloud/deployments-destroying.png[The deployment is be
 
 After the deployment is completely removed, it will have a "Destroyed" state on the [breadcrumb]#Deployments# tab. At this point you will no longer be charged for this deployment.
 
+=== Migrating a Deployment
+
+You can migrate your user data to a FusionAuth Cloud deployment or from FusionAuth Cloud to a self-hosted FusionAuth instance. 
+
+If you are interested in migrating from an auth system other than FusionAuth, you're looking for our link:/docs/v1/tech/guides/migration/[Migration Guide].
+
+==== Migrating To FusionAuth Cloud
+
+To migrate from a self-hosted instance to FusionAuth Cloud:
+
+* You must be running PostgreSQL. If you are running MySQL, please migrate to a PostgreSQL database locally. Here's a link:https://fusionauth.io/community/forum/topic/985/migrating-from-mysql-to-postgresql[forum post on how to do this]. This has been provided by the community and is not supported nor endorsed by FusionAuth.
+* Export your FusionAuth database to a file.
```

Comment:
This might be too specific but is helpful if the DB is large.


-------------------

When you run the `pg_dump` command, please use the following options: `--format=c` and `--compress 9`

-------------------

Reply:
These options compress the database export as much as possible.